packages/Kokkos/src/DTK_PointCloud_def.hpp

```diff
@@ -105,8 +104,8 @@ class LocalBoundingBoxFunctor
     {
         for ( size_t d = 0; d < 3; ++d )
         {
-            dst[d] = std::numeric_limits<SC>::max();
-            dst[d + 3] = -std::numeric_limits<SC>::max();
+            dst[d] = KokkosHelpers::numericLimitsMax<SC>();
+            dst[d + 3] = -KokkosHelpers::numericLimitsMax<SC>();
```

Comment:
Can't you just use `Kokkos::Details::ArithTraits<SC>::max()` from `Kokkos_ArithTraits.hpp`?